hs_core/hydroshare/utils.py

```diff
@@ -459,6 +463,8 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
         if value:
             if avu_name == 'isPublic' and set_to_private:
                 istorage.setAVU(dest_coll, avu_name, 'False')
+            elif avu_name == 'quotaUserName' and requesting_user:
+                istorage.setAVU(dest_coll, avu_name, requesting_user.username)
```

Comment:
@alvacouch This routine copy_resource_files_and_AVUs are specifically called by our resource coping and versioning. For example, if you copy a public resource created by others, you will own this copied resource, and then you which is the requesting_user for copying a resource, will be changed for quota, right? Does this make sense? Let me know if this is not clear and you have further comments on this.